apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/orderBySelect.module.scss

```diff
@@ -0,0 +1,3 @@
+.orderBySelect {
+  min-width: 15rem;
```

Comment:
Yep, retried it and using this doesn't work:
```css
.orderBySelect {
  width: fit-content;
}
```

Screenshot with the above styling after first selecting the shortest string in the dropdown (i.e. "Lähimmät ensin") and then opening it:
![dropdown](https://user-images.githubusercontent.com/77663720/211816177-4a7d6fd6-a89e-4ada-a2fd-4f749a302410.png)

Reply:
@melniiv Any tips on how to make the dropdown adjust to its widest content?

Reply:
Asked for help on this in a [thread](https://helsinkicity.slack.com/archives/CHCV3KTHA/p1673861198693199) on #designsystem.

Reply:
There's now a ticket [HDS-1550](https://helsinkisolutionoffice.atlassian.net/browse/HDS-1550) ("Select width does not scale according to options width") about the possible improvement to Helsinki Design System to support this use case.

Reply:
After discussing this with HDS personnel I'm going with the hardcoded min-width as there doesn't seem to be any low hanging fruit here so to speak i.e. no really easy way to do it so that the Select component would adjust to the width of its widest selectable string—there was one way discussed to calculate the width of the selectable strings in `ch` metrics and take their maximum and add some padding to it and use that as the Select component's width but that wasn't completely straightforward either because of the need for paddings and `ch` metrics not being accurate for non-monospace fonts so choosing not to try to implement that here.